packages/io/io/src/serialize.js

```diff
@@ -0,0 +1,36 @@
+import { flatten } from '@jscad/array-utils'
+
+import { serializers } from './serializers.js'
+
+/**
+ * Serialize the given objects as per the given mimeType into the external format.
+ * Options can be provided to over-ride or suppliment the defaults used during serialization.
+ * See each serializer package for available options.
+ * @param {Object} options - options used during deserializing
+ * @param {String} mimeType - MIME type of the file
```

Comment:
Should mimeType be one of the options fields?

Reply:
I considered that but thought the options should be more focused on the deserialiing / serializing. For example, segments for deserilizing, or unit for serializing.